.github/workflows/gh-pages.yml

```
@@ -0,0 +1,31 @@
+name: github pages
+
+on:
+  push:
+    branches:
+      - v2  # Set a branch to deploy
+  pull_request:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true  # Fetch Hugo themes (true OR recursive)
+          fetch-depth: 0    # Fetch all history for .GitInfo and .Lastmod
+
+      - name: Setup Hugo
+        uses: peaceiris/actions-hugo@v2
+        with:
+          hugo-version: 'latest'
+          extended: true
+
+      - name: Build
+        run: hugo --minify
+
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./public
```

```suggestion
          publish_dir: ./public
          cname: openforcefield.org
```
I think we need this

---

We need a way to test this since if we YOLO it we might take the website down. I will investigate.

---

Thanks!